docs/src/release_notes/v1.20.md

```diff
@@ -0,0 +1,55 @@
+# Release notes for CloudNativePG 1.20
+
+History of user-visible changes in the 1.20 minor release of CloudNativePG.
+
+For a complete list of changes, please refer to the
+[commits](https://github.com/cloudnative-pg/cloudnative-pg/commits/release-1.20)
+on the release branch in GitHub.
+
+## Version 1.20.0
+
+**Release date:** April 27, 2023
+
+!!! Important "Important changes from previous versions"
+    CloudNativePG 1.20 introduces some changes to the default behavior of a
+    few features, compared to previous versions of the operator. The goal of
+    these changes is to improve the resilience of a Postgres cluster out of the
+    box through convention over configuration. For clusters with one or more
+    replicas:
+    
+      - Replication slots for high availability are now enabled by default,
+        unless explicitly disabled
+      - Backup from standby is now enabled by default, unless `target` is
+        explicitly set to `primary`
+      - Restart of the primary is now the default method to complete the
+        unsupervised rolling update procedure (`primaryUpdateMethod`
+        defaults to `restart`, unless explicitly set to `switchover`)
+    
+    Finally, the `-any` service is now disabled by default. For further information,
+    please refer to the ["Installation and upgrades" section](installation_upgrade.md).
```

Comment:
The `-any` service disabling is backported to 1.18 and 1.19, so in this case there's no difference between 1.20 and the other releases.
Since we do mention this in the Enhancements section, I would drop it from herem